controllers/drpolicy_controller.go

```diff
@@ -241,9 +241,22 @@ type drpolicyUpdater struct {
 
 func (u *drpolicyUpdater) deleteDRPolicy(drclusters *ramen.DRClusterList,
 	secretsUtil *util.SecretsUtil,
-	ramenConfig *ramen.RamenConfig) error {
+	ramenConfig *ramen.RamenConfig,
+	mwu *util.MWUtil) error {
```

Comment:
We do not need mwu here, we can use secretsUtil client as well. As there is not manifest work in progress in this function.

Technically secretUtil can also be avoided and formed within this function, but that is a separate concern.